src/unk_020553DC.c

```diff
@@ -119,23 +119,23 @@ void sub_020553DC()
     return;
 }
 
-void sub_020553F0(FieldSystem *fieldSystem, u16 param1)
+void Sound_SetSpecialBGM(FieldSystem *fieldSystem, u16 param1)
```

Comment:
**suggestion:** I think `param1` here is a SDAT ID. For all the invocations of this function that pass a constant, you should be able to find a matching constant in the SDAT consts file. I would recommend just ctrl+Fing for the number in the generated file `build/consts/sdat.h`.